yhaxen.json

```diff
@@ -85,7 +91,8 @@
 			[
 				"-cp", "${variable:sourceDirectory}",
 				"--macro", "include('yloader')",
-				"-js", "none", "--no-output"
+				"-js", "none", "--no-output",
+			  	"${dependency:*:classPath:-cp}"
```

Comment:
have this first in the array, project sources `${variable:sourceDirectory}` must be defined after dependency sources `${dependency:*:classPath:-cp}`